docs/ums/README.md

```diff
@@ -123,16 +131,16 @@ export UMS_TOKEN=`curl -X POST -d '{
 </CodeSwitcher>
 
 ## Fetch a User
-To fetch an user you can look at the following methods:
+To fetch a user you can look at the following methods:
 
 <CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 // via alias
-const user = await ums.getUser('Elon Musk'); 
+getUser('Elon Musk').subscribe(user => console.log(user)); 
 // via email 
-const user = await ums.getUser('elon@tesla.com'); 
+await ums.getUser('elon@tesla.com').subscribe(user => console.log(user));
```

Comment:
Why it is a different approach with async and without?

Reply:
Just forgot to remove `await`, fixed